### PR TITLE
feat: Time zone changeable

### PR DESCRIPTION
**Pull request by kawamataryo**
Hi. Thank you for very useful GitHub Actions.

I wanted to set the merge time in my time zone.
So  I added option to be able to change the time zone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ jobs:
           merge_method: squash
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Set your time zone. Default is UTC
+          TIME_ZONE: 'UTC'
```
Comment (gr2m) [inline README.md:33]: could you set it as `with` parameter instead? You will need to update [inputs in `action.yml`](https://github.com/gr2m/merge-schedule-action/blob/e76675203a2b4eef81ae5be434b4614ab1afbeb9/action.yml#L9-L15)

Comment (kawamataryo) [reply README.md]: Thank you your review.
I corrected them.

6d3a429

**Review by gr2m — APPROVED** [review]
Great, thank you!

**Pull request merged**